src/applications/mhv-medications/tests/e2e/pages/MedicationsRefillPage.js

```diff
@@ -121,10 +121,11 @@ class MedicationsRefillPage {
 
   clickMedicationsLandingPageBreadcrumbsOnRefillPage = () => {
     cy.get('[data-testid="rx-breadcrumb"]').should('be.visible');
-
-    cy.get(`[href="${medicationsUrls.MEDICATIONS_ABOUT}"]`).click({
-      waitForAnimations: true,
-    });
+    cy.get('[data-testid="rx-breadcrumb"]')
```

Comment:
Note: More specific test looking at the link inside the breadcrumbs as it was also matching the link in the sec nav